manifests/machineconfigcontroller/deployment.yaml

```diff
@@ -27,31 +27,26 @@ spec:
             cpu: 20m
             memory: 50Mi
         terminationMessagePolicy: FallbackToLogsOnError
-      - name: oauth-proxy
```

Comment:
Looks like there are some unit testing failures, e.g. `TestRenderAsset` is still explicitly referencing OauthProxy